### ISSUE TITLE
Add "public" account

### ISSUE THREAD

**Issue by savikko** [savikko/pinnikey#13]
There should be one "public" account for each dropzone/dropzone desktop.

There may be environments where users want to freely manifest people to loads, mainly used on small dropzones which are quite autonomic.

**Comment by AlexandrePirojenko** [comment]
While it is important to be able to manifest fellow jumpers, it is not a good idea to give any general account the power to manifest. Instead I think there should be a team account. Team could consist of regular jumpers and could be created even for one jump only. Team account could be for example students and an instructor. Should work like this: create a team, invite people, wait for response, manifest the whole group.

**Comment by savikko** [comment]
I agree, but there could be also the option to make one master account just for manifesting people to load. Of course there should be possibility to authenticate person somehow (Person 1 manifested by Person 2)..

**Comment by AlexandrePirojenko** [comment]
Perhaps a role of "girl at the manifest" could be given to an account. On small DZ:s there could be a virtual jumper with such a role logged in to a computer at all times. I just don't think it has to be special or public, just another account (with the role). But I still think the team should eliminate any problem with manifesting. If a jumper manifests himself to the team, he automatically gives any team member the right to manifest him.
